### ISSUE TITLE
non-supervisor install?

### ISSUE THREAD

**Issue by nickelnine** [diyhue/hassio-addon#14]
I'm new to diyHue and would love to get it set up in my Home Assistant instance but how can I do so using the HA Container installation?

Is there a HACS method of installing it?

**Comment by Mevel** [comment]
I guess you have ha in a docker instance running and not supervised or via ha os.


Just use the docker install method for diyhue to install it alongside home assistant.

Diyhue.org/getting-started

**Comment by nickelnine** [comment]
Yes, running HA in docker. I installed diyHue using this command: 
`sudo apt update && sudo apt upgrade -y`
`curl -sSL https://get.docker.com | sh`
`sudo usermod -aG docker pi`
`sudo docker run -d --name diyHue --restart=always --network=host -e MAC=XX:XX:XX:XX:XX:XX -v /mnt/hue-emulator/export:/opt/hue-emulator/export diyhue/core:latest`

I'm lost now on locating the `config.json` to make the necessary changes.

**Comment by hendriksen-mark** [comment]
its not recommended to change the config.yaml, if you want to change something do this via the webui or hue app.
your config file should be in the dir you specify in the docker command `/mnt/hue-emulator/export`
please close this issue if the is done.
